packages/overlay/src/vaadin-overlay.js

```diff
@@ -214,22 +185,13 @@ class Overlay extends ThemableMixin(DirMixin(ControllerMixin(PolymerElement))) {
        */
       renderer: Function,
 
-      /**
-       * The template of the overlay content.
-       * @type {HTMLTemplateElement | null | undefined}
-       */
-      template: {
-        type: Object,
-        notify: true,
-      },
-
       /**
        * References the content container after the template is stamped.
        * @type {!HTMLElement | undefined}
+       * @deprecated The content container is now the overlay itself.
        */
       content: {
         type: Object,
-        notify: true,
```

Comment:
It is not clear why it is decided to keep this property but disable notifying. Could you clarify this, please?

Reply:
I think we could remove it altogether instead of deprecating, it's not really needed.

Reply:
Submitted #4758 to remove usage of `overlay.content` in other components, please check it out.

Reply:
Removed the `content` property altogether and updated remaining tests where it was used.